### PR TITLE
Feature/mount alert with text field

### PR DESCRIPTION
**Pull request by CibelePaulinoAndrade**
Create custom alert neon with text field

Adjusting Labels Default -> From now on, no changes to text alignment in the default labels file should be allowed.

### CODE REVIEW

File: FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+//  CustomAlertNeonTextFieldViewController.swift
+//  FinalChallenge
+//
+//  Created by Cibele Paulino on 28/10/19.
+//  Copyright © 2019 Elias Paulino. All rights reserved.
+//
+
+import UIKit
+
+class CustomAlertNeonTextFieldViewController: UIViewController {
```
Comment (biaplutarco) [inline FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift]: Its a name too big, the word "custom" isn't really necessary. You could change to TextFieldAlertViewController

File: FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    var message: String
+    var exampleTextField: String
```
Comment (biaplutarco) [inline FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift]: Why "example"?

File: FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        let view = NeonFrameView.init(titleView: nil,
+                                      containerView: alertView,
+                                      withCloseButton: true,
+                                      withTitle: false)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.delegate = self
+        return view
+    }()
+    
+    lazy var alertView: CustomAlertNeonTextField = {
```
Comment (biaplutarco) [inline FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift]: The name alertView looks like it is a view, but it is a TextField instead. Perhaps you could change to a name more easy to understand what it is

File: FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        let alert = CustomAlertNeonTextField.init(message: message, exampleTextField: exampleTextField)
```
Comment (biaplutarco) [inline FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift]: omg, now I understand. Please, when you are giving a name to a class use the name of what that class is as a suffix. For exemple: if I'm creating an table view cell, I will call it MyTableViewCell. In that case, I was thinking that a CustomAlertNeonTextField it was an TextField. 
(sry for my English)

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+//  FinalChallenge
+//
+//  Created by Cibele Paulino on 28/10/19.
+//  Copyright © 2019 Elias Paulino. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CustomAlertNeonTextField: UIView {
```
Comment (biaplutarco) [inline FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift]: Please, change this name its confuse

Comment (biaplutarco) [reply FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift]: Could be: TextFieldAlertView or AlertWithTextFieldView

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    var message: String
+    var exampleTextField: String
```
Comment (biaplutarco) [inline FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift]: Ok, exampleTextField has the same problem that I told you above. It is a String but have TextField as a Suffix, so its confusing. And for I notice this var its the String of the placeholder for the UserInputTextField. You can just call it placeholder or something like this.

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+    var activeField: UITextField?
```
Comment (biaplutarco) [inline FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift]: When you are using this?

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextFieldDelegate.swift [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+//
+//  Created by Cibele Paulino on 28/10/19.
+//  Copyright © 2019 Elias Paulino. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+/// Delegate  used for custom alert neon TextField to comunicate to another component
+protocol CustomAlertNeonTextFieldDelegate: class {
```
Comment (biaplutarco) [inline FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextFieldDelegate.swift]: Don't forget to uptade the name if you change the class's name

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextFieldDelegate.swift [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    /// Function that communicates about the touch of the save button in the completed task view
+    func alertSaveButtonHasBeenTouched()
```
Comment (biaplutarco) [inline FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextFieldDelegate.swift]: Your view it's called Alert pipipopo and in your view has only one saveButton, so I guess the name "alert" it's just make the method name longer then its necessary

File: FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift (hunk 1 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift]: done

File: FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift (hunk 2 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift]: I switched to textFieldPlaceHolder

File: FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift (hunk 3 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift]: But it's the alert view ... it's a view

File: FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift (hunk 4 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Controllers/Alerts/Custom Alert Neon With TextField/CustomAlertNeonTextFieldViewController.swift]: kkkkkkkk ...... okay

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift (hunk 5 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift]: done

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift (hunk 6 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift]: done

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift (hunk 7 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextField.swift]: i`m not, i wil remove... this was used for test only

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextFieldDelegate.swift (hunk 8 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextFieldDelegate.swift]: done

File: FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextFieldDelegate.swift (hunk 9 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Views/Custom Alert Neon With TextField View/CustomAlertNeonTextFieldDelegate.swift]: done

**Pull request merged**